openfecwebapp/templates/layouts/main.html

```diff
@@ -16,6 +16,7 @@
   {% endif %}
   {% if api_location %}
   <script>
+    ANALYTICS = {{ use_analytics|json }};
```

Comment:
Some discussion in the slack channel on this, but the net is
- We have a custom `json` template filter, we'll remove/replace it with `tojson` in the next sprint.
- `safe` isn't necessary but couldn't hurt as this is a boolean value there would never be anything for jinja to accidently escape.
- The `json` filter converts from python boolean `False` to javascript `false`